soc/intel/intel_adsp/Kconfig.defconfig

```diff
@@ -33,14 +33,6 @@ config XTENSA_UNCACHED_REGION
 
 endif # XTENSA_RPO_CACHE
 
-config CONSOLE
-	def_bool y
-
-if CONSOLE
-config WINSTREAM_CONSOLE
-	def_bool y
-endif
-
```

Comment:
@lyakh Not sure these are unused. I think these are the primary logging method used in Intel Zephyr CI. @anasnashif @teburd do you know if this will break something on the on-device tests?